requirements/ci.txt

```diff
@@ -11,7 +11,6 @@ amqp==5.1.1
 appdirs==1.4.4
```

Comment:
I see that the requirements were compiled with python3.10 (`This file is autogenerated by pip-compile with Python 3.10` at the top of the file), I'm not sure if it matters, but it might be worth checking

Reply:
Good idea! I've recompiled dependencies with Python 3.11